estilos/banner.css

```diff
@@ -0,0 +1,181 @@
+* {
+  box-sizing: border-box;
+}
+
+h1 {
+  text-align: center;
+}
+
+body {
+  background-color: orange;
+  padding: 10px;
+  margin: 0;
+}
+
+/*Everything related to this label "a"*/
+a {
+  align-self: center;
```

Comment:
It only works if every anchor tag has the flex property. The best approach is to apply it only in specific cases. Or you could use a new class for buttons